user_docs/en/userGuide.md

```diff
@@ -3861,6 +3872,30 @@ Once a Remote Access session is active, you can switch between controlling the r
 | Send `control+alt+delete` | None | Sends `control+alt+delete` to the controlled computer. |
 <!-- KC:endInclude -->
 
+## Local Image Description {#LocalCaptioner}
+
+NVDA supports generating image descriptions using a local model. This feature allows NVDA to describe image objects encountered during navigation. It supports shortcut key activation, and the result is automatically copied to the clipboard.
+
+Note:
+To reduce the NVDA installer size, the image captioning model is not bundled with NVDA by default.
+
+
+### Getting Started {#LocalCaptionerGettingStarted}
+
+Once the model is ready, press the default shortcut `NVDA+Windows+,` to recognize the image currently navigated by NVDA.
+
+### AI Image Descriptions Key Commands Summary {#LocalCaptionerGestures}
+
+<!-- KC:beginInclude -->
+
+| Name | Key | Description |
+| ------------------------------------- | ---------------------- | ------------------------------------------------------------------------------------------ |
+| get an AI generated image description | `NVDA+Windows+,` | Use a local model to generate a description of the image object currently navigated by NVDA. |
+| toggle image captioning | Unassigned | load/unload the model from memory. |
+| Shows the AI image descriptions settings | Unassigned | Opens the Local Image Description settings panel. |
```

Comment:
please make sure the names here match the final ones in the gestures. Please remove the references to models